content/id/docs/reference/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Referensi
+linkTitle: "Referensi"
+main_menu: true
+weight: 70
+content_type: concept
+---
+
+<!-- overview -->
+
+Bagian dari dokumentasi Kubernetes ini berisi referensi-referensi.
+
+<!-- TODO: translate the rest -->
```

Comment:
nit: this comment is no longer needed you can remove it

Reply:
Sure. Actually I added it there, because there should be some content for it in the English version. I'm planning to translate them later.

Reply:
oh I thought this is coming from the part 1, sure in that case you can leave it.

Reply:
Currently there is no _index.md in the reference page which breaks the sidebar menu, this addition fixes that issue.

Reply:
sounds good 👍